Godeps/Godeps.json

```diff
@@ -435,82 +462,82 @@
 		},
 		{
 			"ImportPath": "github.com/vmware/govmomi",
-			"Comment": "v0.1.0-454-g9051bd6",
+			"Comment": "v0.6.2",
```

Comment:
Yeah, exactly.  `godep` has some very erratic behavior sometimes so it seems kinda wrong to me to modify these vmware lib parameters when we just want to bump the aws driver.